src/BccCode.Linq/QueryProvider/ApiQueryProvider.cs

```diff
@@ -739,10 +761,105 @@ protected override Expression VisitMethodCall(MethodCallExpression node)
                         _where.Append("}}");
 
                         return Expression.Empty();
+                    case nameof(object.ToString):
+                    {
+                        if (node.Arguments.Count != 0)
+                        {
+                            throw new NotSupportedException(
+                                "object.ToString method call is currently only supported in a Where clause without arguments");
+                        }
+
+                        Debug.Assert(node.Object != null);
+                        var objectType = node.Object.Type;
+                        if (objectType.IsGenericType && objectType.GetGenericTypeDefinition() == typeof(Nullable<>))
+                        {
+                            objectType = Nullable.GetUnderlyingType(objectType);
+                            Debug.Assert(objectType != null);
+                        }
+
+                        if (TypeHelper.IsNumberType(objectType))
+                        {
+                            throw new NotSupportedException(
```

Comment:
Shouldn't these be supported, ideally using "InvariantCulture" in .Net?

Reply:
I suggest to leave it for now in the first release. When it is required, it can be implemented later on.